src/tracks/playabletrack/wavetrack/ui/SpectrumView.h

```diff
@@ -22,6 +68,7 @@ class SpectrumView final : public WaveTrackSubView
 
 public:
    using WaveTrackSubView::WaveTrackSubView;
```

Comment:
Maybe delete this using directive for an inherited constructor, now that SpectrumView has its own constructor that overloads it.